yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -281,13 +297,13 @@ const REPORT_PATH_NOTE = "To automatically apply the recommendations, continue t
 func generateBulkAssessmentReport(dbConfigs []AssessMigrationDBConfig) error {
 	log.Infof("generating bulk assessment report")
 	for _, dbConfig := range dbConfigs {
-		assessmentReportPath := dbConfig.GetAssessmentReportPath()
+		assessmentReportPath := dbConfig.GetAssessmentReportBasePath()
```

Comment:
Let's change the variable names here and on line 309 to indicate that it's only the base path. I was confused first..
Let's also add a comment to indicate that the full path will be set by the json/html report functions

Reply:
makes sense